pkg/cosign/verify.go

```diff
@@ -873,7 +884,9 @@ func CheckExpiry(cert *x509.Certificate, it time.Time) error {
 	return nil
 }
 
-func VerifyBundle(ctx context.Context, sig oci.Signature, rekorClient *client.Rekor) (bool, error) {
+// This verifies an offline bundle contained in the sig against the trusted
```

Comment:
nit: what's the key in the map.

Reply:
Note, I only jotted it down (or tried to) in the public methods.

Reply:
This is a good point. That suggests to me it's worth wrapping the `map[string]RekorPubKey` type into a newtype or struct

Reply:
> map[string]RekorPubKey type into a newtype or struct

Right: I'm not even sure if it's the right type to expose to the `CheckOpts` struct, since it's fairly opaque to users. Thinking about this again, maybe it's more worth it to make a more user-friendly constructor for this type? Would address Villie's comment above on repeated code. Lemme try that out

Reply:
done